data/pools/transactionPool.go

```diff
@@ -498,22 +496,10 @@ func (pool *TransactionPool) Lookup(txid transactions.Txid) (tx transactions.Sig
 
 // OnNewBlock excises transactions from the pool that are included in the specified Block or if they've expired
 func (pool *TransactionPool) OnNewBlock(block bookkeeping.Block, delta ledgercore.StateDelta) {
-	var stats telemetryspec.ProcessBlockMetrics
 	var knownCommitted uint
-	var unknownCommitted uint
+	var stats telemetryspec.AssembleBlockMetrics
 
 	committedTxids := delta.Txids
-	if pool.logProcessBlockStats {
-		pool.pendingMu.RLock()
-		for txid := range committedTxids {
-			if _, ok := pool.pendingTxids[txid]; ok {
-				knownCommitted++
```

Comment:
maybe leave these two

Reply:
I agree that we should probably leave this and based on other feedback I'm putting `EnableProcessBlockStats` config back. The open question here is should we leave this block for both `EnableAssembleMetrics` and this or just for the former. 

The concern with enabling for both is that it increases pool locking for all relays emitting AssembleBlock which is many more than currently have ProcessBlockStats enabled